force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -291,7 +337,10 @@ public with sharing class VariablesDataTypesOperators {
      * @return A Decimal representing the monthly paycheck.
      */
     public static Decimal calculateMonthlyPaycheck(Decimal hourlyRate, Double numberOfHours) {
-        return null; // Replace null with the variable you used to store the result
+
+        Decimal weeklyPay = hourlyRate * numberofHours;
+        Decimal monthlyPay = weeklyPay * 4;
```

Comment:
Instead of having a static number 4, try creating a variable and using the variable instead